rq_scheduler/scheduler.py

```diff
@@ -373,6 +373,7 @@ def run(self, burst=False):
                 start_time = time.time()
                 if self.acquire_lock():
                     self.enqueue_jobs()
+                    self.remove_lock()
```

Comment:
The lock should only be removed when the scheduler is run in burst mode. Could you please move this to line 379?

Reply:
I discovered that 10s is hardcoded lock expiration time. We want to set the interval to 5s, but no one deletes the lock, and the scheduler will just say 'Lock already taken - skipping run' for every other loop iteration. Please either accept this PR or introduce custom lock expiration time setting.

Reply:
@selwin really not sure what issue you're perceiving with the acquire_lock

With a single scheduler I observe problems with this code.
The acquire_lock sets a lock in redis for interval+10.
So the first poll loop is fine.
On the second poll, the redis lock is still set (because of +10) - so we don't enqueue_jobs.
On the third poll, the redis lock has expired, we enqueue jobs
on the fourth poll, the redis lock is still present - we don't equeue jobs
on the first poll, the redis lock has expired, we enqueue jobs.
You get the picture.

I think the remove lock is correct with or without burst mode, post the enqueue_jobs(). 

What am I missing?